scripts/sanitycheck

```diff
@@ -1596,18 +1616,31 @@ class TestSuite:
 
         return self.goals
 
+    def run_report(self, filename):
+        with open(filename, "at") as csvfile:
```

Comment:
@nashif did you mean the "a" here for appending? It keeps surprising me. If yes I'll amend --export-tests --help which doesn't mention it.

Reply:
@nashif's answer: yes he meant this. Will update the --help